### PR TITLE
Adds components to the project

### PR DESCRIPTION
**Pull request by HabibPayenda**
### **The following requirements were added to the project:**

- Initialize React app.
- The building blocks of your app should be set as reusable components. Create a directory for your components.

- - Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:
- Books - the default view:
- Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet).
- Create a component called Book for displaying a single book (receive a title and an author as a prop) and reuse it in a component that displays a list of books.
- Should have a form for adding a book (no functionality yet).
- Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).
- Categories:
- Should display a button "Check status" only.

Please check if I have missed something.

**Thank you!**

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const Book = ({ title, author }) => (
+  <div className="bookCard">
+    <p className="title">
+      {title}
+    </p>
+    <p className="author">
+      {author}
+    </p>
+  </div>
+);
```
Comment (OLIPLICHE) [inline src/components/Book.js]: ## Please make sure it has a button for removing books:
- The requirements say that the App has a button for removing books and the button wouldn't remove the books yet, this is just a placeholder for now

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  <header className="header">
+    <p className="logo">
+      BookStore CMS
+    </p>
+    <nav className="nav">
+      <Link to="/">Books</Link>
+      <Link to="categories">Categories</Link>
+    </nav>
+  </header>
+);
```
Comment (OLIPLICHE) [inline src/components/Header.js:16]: - Congratulations 🥇 
- 
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @HabibPayenda , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mateo951 — APPROVED** [review]
### Hi @HabibPayenda, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**